### DISCUSSION TITLE
cmake error while build CXX object pair_nequip.cpp.o

### DISCUSSION THREAD

**Discussion by ghost** [mir-group/pair_nequip#37] [Q&A] [answered]
Good time of a day! I encountered the problem while compiling lammps with pair_nequip. Is there any way fo fix it? 

/home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp: In member function ‘virtual void LAMMPS_NS::PairNEQUIP::init_style()’:
/home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp:97:33: error: ‘int LAMMPS_NS::NeighRequest::half’ is protected within this context
   neighbor->requests[irequest]->half = 0;
                                 ^~~~
In file included from /home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp:26:
/home/common/AI/guskov.ta/soft/lammps/src/neigh_request.h:55:7: note: declared protected here
   int half;    // half neigh list (set by default)
       ^~~~
/home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp:98:33: error: ‘int LAMMPS_NS::NeighRequest::full’ is protected within this context
   neighbor->requests[irequest]->full = 1;
                                 ^~~~
In file included from /home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp:26:
/home/common/AI/guskov.ta/soft/lammps/src/neigh_request.h:56:7: note: declared protected here
   int full;    // full neigh list
       ^~~~
/home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp:101:33: error: ‘int LAMMPS_NS::NeighRequest::ghost’ is protected within this context
   neighbor->requests[irequest]->ghost = 0;
                                 ^~~~~
In file included from /home/common/AI/guskov.ta/soft/lammps/src/pair_nequip.cpp:26:
/home/common/AI/guskov.ta/soft/lammps/src/neigh_request.h:69:7: note: declared protected here
   int ghost;           // 1 if includes ghost atom neighbors
       ^~~~~

**Comment by anjohan** [comment, marked as answer]
Hi,

You currently need to use a slightly older version of LAMMPS, e.g. `stable_29Sep2021_update2`, as specified in the README. This is due to an update to the LAMMPS neighbor list API which we haven't gotten around to following.

Best wishes,
Anders

**Reply by ghost** [reply]
Thanks, that worked
